Gemfile

```diff
@@ -15,7 +15,10 @@ gem "faraday-cookie_jar", "~> 0.0.6"
 
 gem "govuk_sidekiq", "~> 3.0"
 gem "sidekiq-scheduler", "~> 2.2"
-gem "sidekiq-unique-jobs", "~> 5.0"
+# We can't use v5 of this because it requires redis 3 and we use 2.8
+# We use our own fork because the latest 4.x release has a bug with
+# removing jobs from the uniquejobs hash in redis
+gem "sidekiq-unique-jobs", git: "https://github.com/alphagov/sidekiq-unique-jobs", require: false
```

Comment:
Is it worth tagging the current master of alphagov/sidekiq-unique-jobs and referencing it here? Unless you think that lock file will be enough to stop people from upgrading this dependency without checking it? (In the scenario that someone happens to update our fork of the gem because they don't know what it's being used for, or they want to use it in some other way.)

Reply:
Oh, yes. That's a v. good idea. I'll do that and change this PR (and change the publishing-api gemfile again because I already merged it).

Reply:
Created a branch `fix-for-upstream-195-backported-to-4-x-branch` and specified that.  I figure a branch lets us roll it forward more easily rather than a tag in case we find problems with the fix.